### PR TITLE
update standard, run all linters, and update docs

### PR DESCRIPTION
**Pull request by jmkoni**
# What changed, and why?
I might've not followed semver with the last Standard release 😅 . I also wanted to go ahead and update the contributing docs to make it very clear about what needs to pass prior to merging. I've noticed people missing standard and cypress recently :). I went ahead and ran the optional linters as well and this is just what's autofixed, didn't bother to go through and fix any other issues.

### How is this tested? (please write tests!) 💖💪
Ran all the tests! But no new functionality...

### CODE REVIEW

File: app/views/case_court_reports/index.html.erb [hunk 1]
```diff
@@ -12,10 +12,10 @@
     <%= form_with do |form| %>
       <div class="field form-group">
         <%= label_tag :case_number, "Case Number:" %>
-        <% select_options = @assigned_cases.map { |casa_case|
```
Comment (jmkoni) [inline app/views/case_court_reports/index.html.erb:15]: Looks like someone's editor doesn't remove trailing whitespace 🤷‍♀️

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ If you don't have any questions, the issue is clear, and no one has commented sa
 
 Here are the basic steps to submit a pull request.
 
-1. Claim an issue on [our issue tracker][issues] by commenting on the issue saying you are working on it. If the issue doesn't exist yet, open it.
+1. Claim an issue on [our issue tracker][issues] by commenting on the issue saying you are working on it. If the issue doesn't exist yet, open it. Please only claim one at a time.
```
Comment (compwron) [inline doc/CONTRIBUTING.md:11]: 👍

File: doc/CONTRIBUTING.md [hunk 3]
```diff
@@ -20,6 +20,12 @@ Here are the basic steps to submit a pull request.
 
 1. Make the test pass.
 
+1. Run linters and fix any linting errors they brings up.
+   1. `bundle exec standardrb --fix` is required by CI
+   1. But you should also be a good citizen and run:
```
Comment (compwron) [inline doc/CONTRIBUTING.md:25]: I would leave out the "good citizen" part, we have contributors who are not (USA) citizens :)

**Review by compwron — APPROVED** [review]

File: doc/CONTRIBUTING.md (hunk 3 above)
Comment (jmkoni) [reply doc/CONTRIBUTING.md:25]: Hey, they can still be good internet citizens! Definitely not intending to relate it to US citizenship :)

**Pull request merged**